src/services/upstream/upstream.js

```diff
@@ -120,7 +119,7 @@ export default class UpstreamService {
     const valueEth = valueBn.div(new BigNumber(10).pow(18)).toString(10);
     const transaction = {
       to,
-      data: normalizeHexValue(data),
```

Comment:
This is the changes come from origin/fix/normalize-issue branch which I merge with my branch.

Reply:
Understood. Can you please test your fix without that changes? They seem to be not correct from my point of view. 
Btw, PR request from `fix/normalize-issue` was cancelled by me in favor of your one, because first one wasn't proven to solve the issue. So, it's better if you can confirm this change is required to fix the issue. Thanks!

Reply:
yes, I could do that I will create another PR